app/controllers/concerns/course/activities_concern.rb

```diff
@@ -0,0 +1,20 @@
+module Course::ActivitiesConcern
+  extend ActiveSupport::Concern
+
+  # Load recent activity feeds
+  #
+  # @param [Course] course course which activities are related to
+  # @param [Hash]
+  # @option [Integer] :number The number of activities that will be loaded
+  #
+  # @return [Array<Activity>]
+  def recent_activities(course, number: false)
```

Comment:
What if there are some new requirements that need to show activity feeds in other places...